tun/tcp_offload_linux.go

```diff
@@ -269,11 +269,11 @@ func tcpChecksumValid(pkt []byte, iphLen uint8, isV6 bool) bool {
 type coalesceResult int
 
 const (
-	coalesceInsufficientCap coalesceResult = 0
-	coalescePSHEnding       coalesceResult = 1
-	coalesceItemInvalidCSum coalesceResult = 2
-	coalescePktInvalidCSum  coalesceResult = 3
-	coalesceSuccess         coalesceResult = 4
+	coalesceInsufficientCap coalesceResult = iota
+	coalescePSHEnding
+	coalesceItemInvalidCSum
+	coalescePktInvalidCSum
+	coalesceSuccess
```

Comment:
these and the tcpGROResult const values below never go across the network or to disk or to the kernel, do they? (iota should only be used when the values truly don't matter and renumbering them wouldn't break anything)

Reply:
These are all internal yeah, there are some different buffer &/ state handling semantics for specific cases.